lib/tasks/data_updates.rake

```diff
@@ -5,7 +5,7 @@ namespace :data_updates do
       Rake::Task["data_updates:run"].execute
     else
       # Ensure new code has been deployed before we run our update scripts
-      DataUpdateWorker.perform_in(10.minutes)
+      DataUpdateWorker.perform_in(ENV["DATA_UPDATE_WORKER_DELAY"] || 0)
```

Comment:
A few notes:

- If `ENV["DATA_UPDATE_WORKER_DELAY"]` is set, it will return a string. Does `perform_in` handle that?
- Should we set the name to indicate the unit of time that env var represents, such as `DATA_UPDATE_WORKER_DELAY_SECONDS`?
- Is immediately a good default? The comment on the line above indicates that the new code may not be loaded in Sidekiq yet, which may cause an exception on every DUS deploy.

Reply:
> If ENV["DATA_UPDATE_WORKER_DELAY"] is set, it will return a string. Does perform_in handle that?
Yes

> Should we set the name to indicate the unit of time that env var represents, such as DATA_UPDATE_WORKER_DELAY_SECONDS?

Sure we can do that to be more explicit. 

> Is immediately a good default? The comment on the line above indicates that the new code may not be loaded in Sidekiq yet, which may cause an exception on every DUS deploy.

I should update the comment to indicate that it was meant to handle the Heroku case. Beyond that I think 0 is a good default unless you want some sort of delay.

Reply:
For that last part, I was probably unclear. The PR description mentions DEV and BHC as the only Forem instances hosted on Heroku, but I don't believe that's correct. There have been a few people posting on forem.dev about hosting on Heroku, especially when it came to removing Elasticsearch addons.

Reply:
> The PR description mentions DEV and BHC as the only Forem instances

Those were mentioned in the description bc they are the most well known but they are not the only ones. Lee's Forems are on Heroku as well as AEChive and Cluster Employment. I was planning to give Lee a heads up as well as add the ENV to all of our Heroku Forems which I mentioned in the "Are there any post pre deployment tasks we need to perform?" section. 

I am definitely game to do some sort of slower rollout option. Maybe use a check for Heroku specific ENV variable and if that is present make the default 10.minutes. What do you think about something like that?

Reply:
This sounds super reasonable. I don't _love_ that so much of the Ruby ecosystem revolves around Rails and that so much of the Rails ecosystem revolves around deploying to Heroku, but that's what we've got. 

> which I mentioned in the "Are there any post pre deployment tasks we need to perform?" section.

My bad, I didn't look that far down.